README.md

```diff
@@ -0,0 +1,357 @@
+# Rabbit MQ Topic ACL Authorization plugin
+
+## Index
+
+* [Overview](#overview)
+* [Usage](#usage)
+  * [Deployment](#deployment)
+  * [Administration](#administration)
+  * [Examples of use](#examples)
+* [Development](#development)
+  * [Internal Architecture](#architecture)
+  * [Build](#build)
+  * [Test](#test)
+
+## <a name="overview"/> Overview
+### Description
+This project aims to provide an authorization plugin for Rabbit MQ that will manage the access to RabbitMQ resources
+based on the Routing Keys used to publish and bind queues. The ultimate goal of this plugin is to be used along the
+RabbitMQ MQTT plugin in order to use RabbitMQ as an MQTT broker with ACL-based authorization.
+
+Permissions for each routing key will be defined in Access Control Lists (ACLs) with a format similar to the one used
```

Comment:
Making "Mosquito ACLSs" a link to the piece of doc (I guess that in mosquitto documentation) explaining the syntax could be useful.